force-app/main/default/objects/Customizable_Rollup_Settings__c/fields/RecurringDonationLastNDays__c.field-meta.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>RecurringDonationLastNDays__c</fullName>
+    <defaultValue>31</defaultValue>
```

Comment:
Not sure 31 is a good default value for this.  Most RDs are monthly, and If a customer is using Enhanced RDs there will usually be one Opportunity that has been updated in the last 31 days.  Most of the RDs that will be bypassed with this default will be closed RDs.  If that's the intent, cool, otherwise this is too high for the default incremental mode value.

Reply:
@danielfuller0814, please let me know if you change the Default Value from 31. Thanks!